Makefile.am

```
@@ -225,3 +225,12 @@ EXTRA_DIST += \
   config/stl_hash.m4 \
   mac_build.sh \
   ola.spec
+
+# -----------------------------------------------------------------------------
+
+# Create a flake8 python linter target
+if FOUND_FLAKE8
+.PHONY : flake8
+flake8:
```

I don't know if it's strictly necessary, but we normally list Makefile.am here, like so:
```suggestion
flake8: Makefile.am
```

Which should mean that changes to the config within Makefile.am would cause it to need to be re-run.

---

Done.